src/nn/nn_strategy.py

```diff
@@ -62,6 +70,12 @@ def save(self):
     def get_name(self):
         return self.network.get_name()
 
+    def get_total_games_trained(self):
+        return self.network.total_games_trained
+
+    def get_average_game_think_time(self):
+        return self.network.average_think_time
```

Comment:
I think it's worth considering the average think time per move rather than per game. Talking with Jacob, we would have to start tracking the total number of moves. My main concern is tracking on a per game basis could show one game as having a much longer think time if the game lasts more moves, than if the agent wins/loses quickly.

What if we logged average think time per move for a game, and the number of moves? Then we could always calculate an average think time per game if we wanted it.

Thoughts @jacobsteves @Xavientois

Reply:
Yeah thats a very good point. Might even be cool to track number of moves anyway because then we can see if games get longer or shorter as the network trains

Reply:
Agreed

Reply:
Depending on how granular we want to get with the data, we could also store think-time across all 5000 games and track the move-count over all games instead of individually